src/CheckMirrorTree.java

```diff
@@ -0,0 +1,96 @@
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+public class CheckMirrorTree {
+  public static void main(String[] args) {}
+}
+
+class BinaryTree {
+  private Node root;
+
+  // Check this binary tree is mirror or not
+  public boolean isMirror() {
+    Queue<Node> leftQueue = new LinkedList<>();
```

Comment:
Same suggestions in the other pull request applies here as well - https://github.com/musickiper/interview/pull/8